grails-app/views/location/edit.gsp

```diff
@@ -138,6 +138,7 @@
                                                 id="${locationInstance?.parentLocation?.id}"
                                                 value="${locationInstance?.zone?.id}"
                                                 noSelection="['null': '']"
+                                                from="[]"
```

Comment:
Add this as a default in the tag instead of here (when no location for a given id).

Reply:
:+1:

Reply:
fixed